plugins/@grouparoo/airtable/src/lib/disconnect.ts

```diff
@@ -0,0 +1,8 @@
+import { DisconnectPluginAppMethod } from "@grouparoo/core";
+import { IClient } from "./client/interfaces/iClient";
+
+export const disconnect: DisconnectPluginAppMethod<IClient> = async ({
+  connection,
+}) => {
+  return;
+};
```

Comment:
Rather than have a method that does nothing, just remove it!  Destinations don't need a `disconnect()` method

Reply:
https://github.com/grouparoo/grouparoo/pull/2716/commits/1695abffbcc007a1816850d9d61d9556508dc1f4

Reply:
Actually

> error: Error from Initializer name=@grouparoo/airtable file=/Users/brian/grouparoo/grouparoo/plugins/@grouparoo/airtable/dist/initializers/plugin.js type=initialize stack=Plugin validation failure for "@grouparoo/airtable": both app.methods.connect and app.methods.disconnect are required for a Grouparoo Plugin App

Reply:
Well that's a shame!